codebase/superdarn/src.lib/tk/fitacf.2.5/src/fitacf.c

```diff
@@ -99,7 +99,7 @@ int fill_fit_block(struct RadarParm *prm, struct RawData *raw,
     input->prm.cp=prm->cp;
     input->prm.channel=prm->channel;
     input->prm.offset=prm->offset;  /* stereo offset */
-    if ((input->prm.offset == 0) || (input->prm.channel < 2)) {
```

Comment:
Can you comment on why the removal of offset == 0? Sorry I don't understand this much to begin with :p